src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/UserModel.java

```diff
@@ -11,15 +13,27 @@
     private long id;
 
     private String clientToken;
+    private byte[] random;
+    private SecureRandom secureRandom;
+
 
     @OneToMany
     List<AuthModel> authModelList;
 
     public UserModel() {
+
     }
 
     public UserModel(String clientToken) {
-        this.clientToken = clientToken;
+        this.clientToken = getRandomClientToken(clientToken);
+    }
+
+    public String getRandomClientToken(String token) {
+        secureRandom = new SecureRandom();
+        random = new byte[256];
+        secureRandom.nextBytes(random);
+        token = Base64.encodeBase64String(random);
+        return token;
```

Comment:
Zou don!t need the parameter, please remove and return with the result of Base64.encodeBase64String(random);